### PR TITLE
feat: profile default tab

### PR DESCRIPTION
**Pull request by alantoa**
# Why

- If Created list counts = 0, it should go to the Owned tab directly. 
- If the parmas of the current tab exceeds the range, will get an empty list. 

# How

- do some judgment handing the default tab.
- when the params don't match, enter the default tab.

# Test Plan

check if the Profile tab load works.

### CODE REVIEW

File: packages/app/components/profile/index.tsx [hunk 1]
```diff
@@ -48,20 +49,43 @@ const Profile = ({ address }: { address: string | null }) => {
...
   });
+  /**
+   * defalut tab index.
+   * if Created list = 0, it should go to the Owned section directly.
+   * */
+  const defaultIndex = useMemo(() => {
+    if ((router.query as any)?.tab) return (router.query as any)?.tab;
+    const createdIndex =
+      data?.tabs.findIndex((item) => item.type === "created") ?? 0;
+    const onwedIndex =
```
Comment (intergalacticspacehighway) [inline packages/app/components/profile/index.tsx]: nit: typo

**Review by intergalacticspacehighway — APPROVED** [review]
LGTM ❇️

File: packages/app/components/profile/index.tsx (hunk 1 above)
Comment (alantoa) [reply packages/app/components/profile/index.tsx]: thank you! my bad! 🙏

File: packages/app/components/profile/index.tsx [hunk 2]
```diff
@@ -48,20 +49,43 @@ const Profile = ({ address }: { address: string | null }) => {
...
+  const router = useRouter();
+
   const { width } = useWindowDimensions();
   const isDark = useIsDarkMode();
   const contentWidth = useContentWidth();
   const { data } = useProfileNftTabs({
     profileId: profileData?.data?.profile.profile_id,
   });
+  /**
+   * defalut tab index.
```
Comment (axeldelafosse) [inline packages/app/components/profile/index.tsx]: Typo. `defalut` -> `default`

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

File: packages/app/components/profile/index.tsx (hunk 2 above)
Comment (alantoa) [reply packages/app/components/profile/index.tsx]: Thank you! My bad!

**Pull request merged**